packages/pwa-kit-runtime/src/ssr/server/express.js

```diff
@@ -489,3 +489,30 @@ export const getRuntime = () => {
+                    </g>
+                </svg>
+            </div>
+        </html>
+    `)
+}
```

Comment:
What was the reasoning behind making this `helloWorld` route handler a permanent fixture in the runtime? It doesn't seem like the right place to be putting it, unless you have thoughts about it being used across multiple starters. Even so I think the behaviour is so specific that I don't see it being used too often.

Reply:
No strong feelings. I just wanted the template to be properly minimal and the HTML is trivial to maintain. If you'd prefer  it to live in the template, feel free!

Reply:
I attempted to move that express handler into the app itself but I wasn't able t verify that it works as the dev server starts but I cannot connect to it. This might be a larger issue, maybe we can talk about it when you have time.

Reply:
Ignore the above comment. I was having some technical difficulties with my environment. Nothing a good old restart didn't fix.